Recipes.txt

```diff
@@ -0,0 +1 @@
+ # Signature Latte Recipe
```

Comment:
#1 Review of Joe Git Cafe Project
    For improving user friendly wording, can you change the comment to be more descriptive.